node/npm_import_error.ts

```diff
@@ -0,0 +1,25 @@
+class NPMImportError extends Error {
+  constructor(originalError: Error, moduleName: string) {
+    super(
+      `It seems like you're trying to import an npm module. This is only supported in Deno via CDNs like esm.sh. Have you tried 'import mod from "https://esm.sh/${moduleName}"'?`,
```

Comment:
Btw, extracting the module name with the regular expression and showing it here is a really nice touch! ✨